lib/redis/store/interface.rb

```diff
@@ -6,7 +6,7 @@ def get(key, options = nil)
       end
 
       def set(key, value, options = nil)
-        super(key, value)
+        super(key, value, options || {})
```

Comment:
Definitely a good point! I'll make an issue for a future breaking change to fix this.